src/webui/webapplication.cpp

```diff
@@ -228,6 +229,7 @@ void WebApplication::translateDocument(QString &data)
 
         data.replace(QLatin1String("${LANG}"), m_currentLocale.left(2));
         data.replace(QLatin1String("${VERSION}"), QBT_VERSION);
+        data.replace(QLatin1String("${LIBTORRENT_VERSION}"), LIBTORRENT_VERSION);
```

Comment:
This is not right... you should fetch it with another webAPI, there is `Utils::Misc::libtorrentVersionString` and others.